src/keyTypeHandlers/freighter.ts

```diff
@@ -19,7 +19,10 @@ export const freighterHandler: KeyTypeHandler = {
     }
 
     try {
-      const response = await signTransaction(transaction.toXDR());
+      const response = await signTransaction(
+        transaction.toXDR(),
+        custom && custom.network,
```

Comment:
I think if `custom` is falsy, it'll pass the value of `custom` where you probably want to pass `null` in that  case?

Maybe this should be verified with unit tests?

Reply:
@morleyzhi  This was a good catch. It probably would've been fine in practice, but better to be more clear. We want to pass `undefined` if no network is present to trigger the default param. Add tests as well